qiskit/circuit/quantumcircuit.py

```diff
@@ -1167,7 +1168,7 @@ def draw(self, output=None, scale=None, filename=None, style=None,
                 The search path for style json files can be specified in the user
                 config, for example,
                 ``circuit_mpl_style_path = /home/user/styles:/home/user``.
-                See: :ref:`Style Dict Doc <style-dict-doc>` for more
+                See: :class:`~qiskit.visualization.qcstyle.DefaultStyle` for more
```

Comment:
This likely won't work until we add the default style class to the documentation. Normally you would do this by adding the class to `qiskit/visualization/__init__.py`'s autosummary in the module docstring. If you do that the path will be `qiskit.visaulization.DefaultStyle`.

Reply:
Thanks. Should be done in https://github.com/Qiskit/qiskit-terra/pull/5327/commits/a1fd752ee4e1ba0a2313dbf5161968340fd698a4.